auxx/src/Command/DumpBlockchain.hs

```diff
@@ -149,9 +149,9 @@ applyEpoch path = do
              "difficulty is "+|difficulty|+"")
     result <- runConduitRes $
            C.sourceFile path
-        .| decodeBlockDumpC                            -- get a stream of blocks
-        .| (C.dropWhileC ((/= tip) . view prevBlockL) -- skip blocks until tip
-        >> verifyAndApplyBlocksC True)                -- apply blocks w/ rollback
+        .| decodeBlockDumpC                               -- get stream of blocks
+        .| (C.dropWhileC ((/= tip) . view prevBlockL)     -- skip blocks until tip
```

Comment:
"Until" looks suspicious here. You will drop `tip` itself too. I'd say "skip blocks `[start..tip]`" or something.